mlflow/store/artifact/models_artifact_repo.py

```diff
@@ -169,8 +173,14 @@ def download_artifacts(self, artifact_path, dst_path=None):
         :return: Absolute path of the local filesystem location containing the desired artifacts.
         """
 
+        from mlflow.models.model import MLMODEL_FILE_NAME
```

Comment:
Cannot do a non-local import due to circular reference. We really should migrate constants like these to a separate 'neutral zero-dependency' location, but that is out of scope for this PR.